.github/workflows/docker-push.yml

```diff
@@ -0,0 +1,73 @@
+name: Docker - ghcr Build and Push
+
+on:
+  push:
+    schedule:
+      - cron: "0 10 * * *"
```

Comment:
Why is there a need for a cron? Should pushes to release branches suffice?

Reply:
True, the pushes to release and tagging would suffice moving forward, but how do we trigger the previous images?
The solution  i came up with was using cron flag to trigger a job run that would pickup old tags and release branches.
At least that's how I hope it works.

Reply:
we could remove it of course once legacy images are built